### PR TITLE
Dashboard viewing parties

### LINKED ISSUE #10: Dashboard: Viewing Parties
As an authenticated user,
I should see the viewing parties I have been **invited** to with the following details:
- Movie Title, which links to the movie show page
- Date and Time of Event
- who is hosting the event
- list of friends invited, with my name in **bold**

I should also see the viewing parties that I have **created** with the following details:
- Movie Title, which links to the movie show page
- Date and Time of Event
- That I am the host of the party
- List of friends invited to the viewing party

### PR DESCRIPTION
**Pull request by Trevor-Robinson**
____ Wrote Tests ____ Implemented ____ Reviewed

Neccesary checkmarks:

  - [x] All Tests are Passing
  - [x] The code will run locally
  - [x] Ran RuboCop `bundle exec rubocop --safe-auto-correct`

Type of change

  - [x] New feature
  - [x] Bug Fix

Implements/Fixes:

    description closes #

Check the correct boxes

  - [x] This broke nothing
  - [ ] This broke some stuff
  - [ ] This broke everything

Testing Changes

  - [ ] No Tests have been changed
  - [x] Some Tests have been changed
  - [ ] All of the Tests have been changed(Please describe what in the world happened)

Checklist:

  - [x] My code has no unused/commented out code
  - [x] I have reviewed my code
  - [ ] I have commented my code, particularly in hard-to-understand areas
  - [x] I have fully tested my code

(For Fun!)Please Include a link to a gif of your feelings about this branch

Link:

### CODE REVIEW

File: spec/features/events/new_spec.rb [hunk 1]
```diff
@@ -26,11 +26,6 @@
     expect(page).to have_content(@movie_title)
   end
 
-  it 'should have a field for a name' do
-
-    expect(page).to have_field('name')
-  end
-
```
Comment (abreaux26) [inline spec/features/events/new_spec.rb:33]: @Trevor-Robinson just curious, why did you remove this?

File: app/views/events/new.html.erb [hunk 2]
```diff
@@ -1,13 +1,12 @@
 <h2><%= @movie[:title] %></h2>
 <%= form_with url: events_path, local: :true do |f| %>
   <%= f.hidden_field :api_movie_id, value: @movie[:id] %>
-  <%= f.label :name, "Name:" %>
-  <%= f.text_field :name %>
-  <%= f.label :duration, "Duration:" %>
-  <%= f.number_field :duration %>
+  <%= f.hidden_field :name, value: @movie[:title] %>
+  <%= f.label :duration, "Duration:"%>
+  <%= f.number_field :duration, value: @movie[:runtime], min: @movie[:runtime]  %>
```
Comment (abreaux26) [inline app/views/events/new.html.erb:6]: LOVE THIS!!

File: spec/features/events/new_spec.rb (hunk 1 above)
Comment (Trevor-Robinson) [reply spec/features/events/new_spec.rb:33]: For the display of the viewing parties, it is only suppose to show the name of the movie, and not a separate title. As far as I could tell, the seperate event name field wasn't going to be used anywhere. Probably should have checked with you both before just removing though. Sorry about that.

Comment (abreaux26) [reply spec/features/events/new_spec.rb:33]: Oh okay! No worries at all! I was just curious
